conda.recipe/meta.yaml

```diff
@@ -1,8 +1,7 @@
-{% set data = load_setuptools()%}
 
 package:
-  name: {{data.get('name')}}
-  version: {{data.get('version')}}
+  name: mtq
+  version: 0.7.0
```

Comment:
0.7.0 is already tagged, let's use 0.7.1 to add all the latest fixes you will end up merging soon. Otherwise if people built 0.7.0 in the past, we will have different animals with the same version...